notebooks/en/generate_preference_dataset_distilabel.ipynb

```diff
@@ -0,0 +1,607 @@
+{
```

Comment:
<p>"...two different models available on the Hugging Face Hub..."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/182/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>